bootstrap/service.go

```diff
@@ -25,6 +25,9 @@ var (
 	// ErrExternalKey indicates a non-existent bootstrap configuration for given external key.
 	ErrExternalKey = errors.New("failed to get bootstrap configuration for given external key")
 
+	// ErrConflict indicates that the entity that the client is trying to create already exists.
```

Comment:
Are we going to use this error `ErrConflict` ?
I see in code we are using svcerr.ErrConflict not `ErrConflict`

Reply:
We are not using it. That was an oversight.